Components/BlockImageTextParallax/index.twig

```diff
@@ -0,0 +1,25 @@
+<div class="flyntComponent" is="flynt-block-image-text-parallax">
+  <div class="container centerMaxWidthContainer">
+    <div class="box box--{{ imagePosition }}">
+      <div class="box-inner box-inner--content {{ options.theme }}">
+        <div class="box-copy">
+          {{ contentHtml|e('wp_kses_post') }}
+        </div>
+      </div>
+      <div class="box-inner box-inner--image rellax" data-parallax data-rellax-speed="3" data-relax-percentage="0.5">
```

Comment:
Typo in data Attribute? One is rellax and the other relax?

Reply:
Actually, I just saw these settings are overwritten in the script anyways and should therefore be removed here.